### ISSUE TITLE
Web.Config Settings overrides.

### ISSUE THREAD

**Issue by KevinJump** [KevinJump/uSync8#42]
As per uSync 4 you should beable to override the main settings in the web.config 
https://our.umbraco.com/packages/developer-tools/usync/usync/99117-usync-8-wbconfig-appsettings-overides

**Comment by KevinJump** [comment]
The following can be added via the web.config 

```
    <add key="uSync.FlatFolders" value="true" />
    <add key="uSync.ImportAtStartup" value="true" />
    <add key="uSync.ExportAtStartup" value="false" />
    <add key="uSync.ExportOnSave" value="true" />
    <add key="uSync.UseGuidFilenames" value="true" />
    <add key="uSync.Folder" value="~/uSync/v82" />
    <add key="uSync.DefaultHandlerSet" value="publish" /> 
```

**Comment by KevinJump** [comment]
Fixed in a7d52f67a64b7e51261ed5c5f8636ffa30c5d2bc
